lnwire/features.go

```diff
@@ -61,243 +36,217 @@ const (
 	maxAllowedSize = 32781
 )
 
-// Feature represent the feature which is used on stage of initialization of
-// feature vector. Initial feature flags might be changed dynamically later.
-type Feature struct {
-	Name featureName
-	Flag featureFlag
+// localFeatureNames is a mapping of known connection-local feature bits to a
+// descriptive name. All known local feature bits must be assigned a name in
+// this mapping. Local features are those which are only sent to the peer and
+// not advertised to the entire network. A full description of these feature
+// bits is provided in the BOLT-09 specification.
+var localFeatureNames = map[FeatureBit]string{
```

Comment:
The tradeoff in having this assignment be within `lnwire`, rather than the old `features.go`, is that we're now setting all the known feature bits for the entire _package_,  rather than the the _node_. 

With this approach, all the context is pushed into the package, rather than the node that's using the package to advertise their feature bits.